.pre-commit-config.yaml

```diff
@@ -0,0 +1,70 @@
+repos:
```

Comment:
We should add `---` at the beginning of this document in order to address the issue:

1:1 [document-start] missing document start "---"